js/main.js

```diff
@@ -176,3 +182,48 @@ $formEntry.addEventListener('submit', function (event) {
   $entryImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   viewSwapping('entries');
```

Comment:
Before you swap to the entries view, you should make an entry for the entry just saved and add it to the DOM. You have a handy dandy function named `renderEntry` that you can make use of 👍